src/pages/Regions/Region.tsx

```diff
@@ -204,7 +202,8 @@ const Region = () => {
       setDocumentsCount(regionResponse.data.documentsCount);
 
       setPhotosLoading(true);
-      setSixAdmins(regionAdministrationResp.data, 7);
+      setAdmins(regionAdministrationResp.data);
+      getSixAdmins(regionAdministrationResp.data, 7);
```

Comment:
what is the magic number 7?

Reply:
it was before me, ask Ivan)

Reply:
But you can stop this endless road to bad code)